alica_ros_turtlesim/alica/etc/turtle2.yaml

```diff
@@ -0,0 +1,64 @@
+Local:
+  ID: 2
```

Comment:
We can have a single yaml now and get rid of ID. The engine will autogenerate id's.